packages/drivers/local-driver/src/localDocumentStorageService.ts

```diff
@@ -37,7 +41,11 @@ export class LocalDocumentStorageService implements IDocumentStorageService {
 		private readonly id: string,
 		private readonly manager: GitManager,
 		public readonly policies: IDocumentStorageServicePolicies,
+		private readonly localDeltaConnectionServer: ILocalDeltaConnectionServer,
+		private readonly resolvedUrl: IResolvedUrl,
 	) {
+		ensureFluidResolvedUrl(resolvedUrl);
```

Comment:
you can make the optional.

1. remove this check
2. check that they are not undefined in the condition below, and throw and error if they are undefined

Reply:
this will then only break in the new scenario